L1Trigger/Phase2L1ParticleFlow/src/puppi/linpuppi_ref.cpp

```diff
@@ -92,6 +107,17 @@ l1ct::LinPuppiEmulator::LinPuppiEmulator(unsigned int nTrack,
   priorPh_[1] = priorPh_1;
   ptCut_[0] = ptCut_0;
   ptCut_[1] = ptCut_1;
+
+  if (useMLAssociation_) {
+    associationGraph_ = tensorflow::loadGraphDef(associationGraphPath_);
```

Comment:
Is there any reason for 
`associationGraph_` and `associationSesh_` not to be data members of `NNVtxAssoc` rather than `LinPuppiEmulator`